articles/native-platforms/ios-objc/10-customizing-lock.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Customizing Lock
+description: This tutorial will show you how to customize the Lock widget UI.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../includes/github', { link: 'https://github.com/auth0-samples/auth0-ios-objc-sample/tree/master/10-Customizing-Lock', }) %>
```

Comment:
@pablolvillar @sebacancinos 
Both the directory and the filename are wrong and this causes the page to fail to load. The correct is `../../_includes/_github `